src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/models/FormLayoutElement.java

```diff
@@ -14,6 +14,7 @@
   private String id;
   private HashMap<String, String> dataModelBindings;
   private TextResourceBindings textResourceBindings;
+  private String optionsId;
```

Comment:
We should also support showing values for components where they have options set up directly on the component. Can be added as an `options` prop on the component itself.

Reply:
my bad.. Added support for options too.